base/compiler/inferencestate.jl

```diff
@@ -112,7 +121,7 @@ mutable struct InferenceState
             ssavalue_uses, throw_blocks,
             Vector{Tuple{InferenceState,LineNum}}(), # cycle_backedges
             Vector{InferenceState}(), # callers_in_cycle
-            #=parent=#nothing,
+            #=parent=#nothing, #= has_opaque_closures =# false,
```

Comment:
Why does the style change from annoating on right (old line 113 and 114) to annotating inline?
```suggestion
            nothing, # parent
            false, # has_opaque_closures
```